include/psa/crypto_values.h

```diff
@@ -883,7 +892,9 @@
      (alg) & PSA_ALG_KEY_DERIVATION_STRETCHING_FLAG)
 
 /** An invalid algorithm identifier value. */
+/* *INDENT-OFF* (https://github.com/ARM-software/psa-arch-tests/issues/337) */
```

Comment:
Rather than make those changes, we should perhaps patch the compliance suite (since we have our own patched branch anyway). But that would be more work involving more moving parts, so for now let's go with a self-contained workaround.